src/components/Ledger/LedgerConnect.js

```diff
@@ -273,6 +272,7 @@ class LedgerConnect extends React.Component<Props, State> {
       if (error instanceof LocalizableError) {
         errMsg = intl.formatMessage({
           id: error.id,
+          // $FlowFixMe
```

Comment:
We can remove this just by removing the `?` in `defaultMessage` prop in `LocalizableError.js`